docs/deploy-options.md

```diff
@@ -2,12 +2,19 @@
 
 The [README][0] shows you a simple way to get started with Contour on your cluster. This topic explains the details and shows you additional options.
 
+## Manual and Specialized Deployments
+
+This document outlines how to run Contour using a Kubernetes Service of `Type: LoadBalancer`.  If you don't have a cluster with that capability or if you don't want to use it you have two other options:
+
+* Use a Kubernetes Service with `Type: NodePort`.  This configure every node in your cluster to forward traffic on that Node to Contour.
+* Don't use a Kubernetes Service at all.  You can run with `hostNetwork: true` on your pod and direct traffic to port 8080 on the nodes that are running Contour. See the [AWS NLB tutorial][3] as an example.
```

Comment:
And then if we keep the current content in its current location, we should move this one down to the section ##Running without ...

Reply:
yup